src/components/ClusterManager.js

```diff
@@ -69,7 +69,15 @@ const styles = {
     marginLeft: '-1rem',
     marginRight: '-1rem',
     borderBottom: `1px solid ${colors.gray[3]}`
-  }
+  },
+  button: (open, disabled) => ({
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    color: disabled ? colors.gray[2] : (open ? colors.blue[1] : colors.blue[0]),
+    borderRadius: '5px 5px 0 0',
```

Comment:
nit: Setting the color at this level means we need to override it again on the text. We could just scope this down to the icon. Also, this border-radius is unnecessary.